modules/network-observability-flow-filter-parameters.adoc

```diff
@@ -38,6 +38,9 @@ The flow filter rules consist of required and optional parameters.
 |`protocol`
 | Defines the protocol of the flow filter rule. Possible values are `TCP`, `UDP`, `SCTP`, `ICMP`, and `ICMPv6`.
 
+|`tcpFlags`
+| Defines the TCP flags to filter flows. Possible values are `SYN`, `SYN-ACK`, `ACK`, `FIN`, `RST`, `PSH`, `URG`, `ECE`, `CWR`, `FIN-ACK`, `RST-ACK`.
```

Comment:
```suggestion
| Defines the TCP flags to filter flows. Possible values are `SYN`, `SYN-ACK`, `ACK`, `FIN`, `RST`, `PSH`, `URG`, `ECE`, `CWR`, `FIN-ACK`, and `RST-ACK`.
```

Reply:
This update looks like it was missed

Reply:
man, I hate when I forget to crtl+s on my VSCode files 🤦‍♀️ always happens with tiny changes.